recipes/aghast/0001-Fix-packaging-for-conda.patch

```diff
@@ -0,0 +1,139 @@
+From d3742537a4160162e8c11a05697cbd6392b5effb Mon Sep 17 00:00:00 2001
```

Comment:
I've opened a PR to remove the need for this in the next release: https://github.com/diana-hep/aghast/pull/15